blackrock/templates/portal/base_portal.html

```diff
@@ -49,7 +49,7 @@
 <body>
     <div id="shieldbox"></div>
     <div id="masthead_top">
-        <a href="https://ccnmtl.columbia.edu/"><span id="ccnmtl"></span></a>
+        <a href="https://www1.ccnmtl.columbia.edu/"><span id="ccnmtl"></span></a>
```

Comment:
I can't get to https://www1.ccnmtl.columbia.edu/. Maybe you mean just http://ccnmtl.columbia.edu/ ?